gBrogue Source/PlatformCode/curses-platform.c

```diff
@@ -217,13 +217,18 @@ static boolean modifier_held(int modifier) {
 	return 0;
 }
 
+static void notify_event(short eventId, int data1, int data2, const char *str1, const char *str2) {
+  //Unused
+}
+
 struct brogueConsole cursesConsole = {
 	gameLoop,
 	curses_pauseForMilliseconds,
 	curses_nextKeyOrMouseEvent,
 	curses_plotChar,
 	curses_remap,
-	modifier_held
+	modifier_held,
+	notify_event
```

Comment:
notify_event is added to the platform struct for all targets, as a no-op except for web. Basically it's used for the base game to send messages to the platform specific code when something interesting happens. It's how web-brogue tracks high scores etc.